src/utils.ts

```diff
@@ -33,3 +35,25 @@ export function parseString(s: string): string | undefined {
 export function parseNumber(s: string): number | undefined {
   return parseInt(s) || undefined
 }
+
+export function parseParameters(parameterOverrides: string): Parameter[] {
+  if (parameterOverrides.trim().startsWith('file://')) {
```

Comment:
What do you think of doing this using the node built in https://nodejs.org/api/url.html and https://nodejs.org/api/fs.html of which `readFileSync` does support `file://`. We could then think also of loading from a `http` url.

Reply:
aa nice, i'll check it out!